.github/workflows/setup/action.yaml

```diff
@@ -1,51 +1,36 @@
-name: 'openpilot env setup'
+name: 'openpilot env setup, with retry on failure'
 
```

Comment:
let's call this setup-with-retry

Reply:
or can we do a nested action like [actions/cache](https://github.com/actions/cache)

Reply:
Renamed to setup-with-retry